src/main/java/zpi/service/AttractionServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package zpi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import zpi.entity.Attraction;
+import zpi.repository.AttractionRepository;
+
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class AttractionServiceImpl implements AttractionService{
+
+    private final AttractionRepository attractionRepository;
+
+    @Autowired
+    public AttractionServiceImpl(AttractionRepository attractionRepository) {
+        this.attractionRepository=attractionRepository;
+    }
+    @Override
+    public Attraction saveAttraction(Attraction attraction) {
+        return attractionRepository.save(attraction);
+    }
+
+    @Override
+    public List<Attraction> fetchAttractionList() {
+        return (List<Attraction>)attractionRepository.findAll();
+    }
+
+    @Override
+    public Attraction updateAttraction(Attraction attraction, Integer attractionId) {
+        Attraction attractionDB
+                = attractionRepository.findById(attractionId)
+                .get();
+
+        if (Objects.nonNull(attraction.getTitle())
+                && !"".equalsIgnoreCase(
```

Comment:
To też jest takie ehh. Lepiej nie sprawdzać wcale albo zrobić walidacje na polach w enitity